os/kernel/Kconfig

```diff
@@ -268,6 +268,13 @@ config PREALLOC_TIMERS
 		pool of preallocated timer structures to minimize dynamic allocations.  Set to
 		zero for all dynamic allocations.
 
+config CONFIG_IDLE_PANIC_TIMER
```

Comment:
Let us change the naming of this config as HANG_DETECT_WAIT_SECS . 
Description: If a task is running for more than HANG_DETECT_WAIT_SECS without a context switch, then we assume that the device has hanged and we try to print the debug logs and restart the device.